resource-timing/resources/TAOResponse.py

```diff
@@ -0,0 +1,30 @@
+def main(request, response):
+    origin = request.GET.first('origin', request.headers['origin'])
+    response.headers.set('Access-Control-Allow-Origin', origin)
+
+    tao = request.GET.first('tao')
+
+    if tao == 'zero':
+    # zero TAO value, fail
+        pass
+    elif tao == '*':
+    # wildcard, pass
+        response.headers.set('Timing-Allow-Origin', '*')
+    elif tao == 'null':
+    # null, fail
+        response.headers.set('Timing-Allow-Origin', 'null')
+    elif tao == 'origin':
+    # case-sensitive match for origin, pass
+        response.headers.set('Timing-Allow-Origin', origin)
+    elif tao == 'space':
+    # space seperated list of origin and wildcard, fail
+        response.headers.set('Timing-Allow-Origin', (origin + ' *'))
+    elif tao == 'multi':
+    # more than one TAO values, fail
```

Comment:
This one should not fail, per HTTP spec multiple headers can be combined into a comma separated list.